client/src/views/Feed/Feed.vue

```diff
@@ -19,12 +19,16 @@
         </div>
         <div class="content-wrapper">
           <div class="heart-and-comments">
-            <div
-              class="heart"
-              @click="updateLikes({ key: key, updates: ({ hasBeenLiked : post.hasBeenLiked, likes: post.likes })  })"
-            >
-              <i class="far fa-heart fa-lg" :class="{'fas': post.hasBeenLiked}"></i>
-              <p class="likes">{{post.likes}}</p>
+            <div class="heart">
+              <i
+                class="far fa-heart fa-lg"
+                :class="{'fas': post.hasBeenLiked}"
+                @click="updateLikes({ key: key, updates: ({ hasBeenLiked : post.hasBeenLiked, likes: post.likes })  })"
```

Comment:
You already have an identical function. DRY.

Reply:
It isn't my func.
It is Roman's temp function and I have already made normal func to interact with backend, I will push it tommorow.